File: Java/BubbleSort.java
import java.util.Arrays;

public class BubbleSort 
{
    public static void main(String [] args) {
        int[] arr = new int[]{9, 8, 7, 6, 5, 5, 4, 3, 2, 1, 0};
        
        int counter = 1;
        for(int i=0;i<arr.length-1;i++) {
            for(int j=0;j<arr.length-counter;j++) {
                if(arr[j+1] < arr[j])
                    arr = swap(arr, j, j+1);
            }
        }
        
        System.out.println(Arrays.toString(arr));
    }
    
    public static int[] swap(int[] arr, int i, int j) {
        int temp = arr[i];
        arr[i] = arr[j];
        arr[j] = temp;
        
        return arr;
    }
}

File: Python/BubbleSort.py
def swap(array, idx0, idx1):
    temp = array[idx0]
    array[idx0] = array[idx1]
    array[idx1] = temp
    return array



arr = [9, 8, 7, 6, 5, 5, 4, 3, 2, 1, 0]

counter = 1

for _ in range(0, len(arr)-1):
    for j in range(0, len(arr)-counter):
        if arr[j+1] < arr[j]:
            arr = swap(arr, j, j+1)
    counter += 1

print(arr)
